content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -2,7 +2,39 @@
 name: Workspace
 match: [workspace]
 tooltip: >-
-  Directory containing all your project files e.g. raw datasets, source code, ML
-  models, etc. Typically, it's also a Git repository. It will contain your DVC
-  project.
+  Directory containing all your DVC project files, e.g. raw data, source code,
+  ML models. One project version at a time is visible in the workspace.  
+  See full [concept page](/doc/user-guide/basic-concepts/workspace).
```

Comment:
Hi @julieg18 and @rogermparent: Is there a way to add a `link` property to the tooltip frontmatter structures? So it links from here:

![image](https://user-images.githubusercontent.com/1477535/126463980-f106bfc0-a02a-42a0-a1b3-16f2df4b8f93.png)

Thanks

Reply:
Ping @julieg18 any thoughts?

Reply:
I would think we'd want to automatically know the link and populate it without manually specifying it, but either way (or both if we want some override behavior) should work. Julie may have more details, since she built out this feature.

Reply:
> automatically know the link and populate it without manually specifying it

OK but the tricky part is that it should only link to the corresponding `user-guide/basic-concepts/<name>` page IF there's MD content in this same file (under the frontmatter). For now a manual `link` field is good enough and it can become the override later if we implement the auto-linking after that.

So should we make that change in this PR?

Reply:
Co-assigned with @julieg18 for now 🙂

Reply:
It should be possible to implement this! Do we want the "Workspace" to still have the same styling as the other tooltip titles(black with an icon that appears on hover) or do we want it to look like a link(blue with underline on hover)?

Reply:
Sounds good. Should look like a link but I guess also bold since it's the entry's title? Up to you